src/Model/Actions.re

```diff
@@ -62,11 +62,9 @@ type t =
   | EditorScroll(EditorId.t, float)
   | EditorScrollToLine(EditorId.t, int)
   | EditorScrollToColumn(EditorId.t, int)
-  | OpenExplorer(string)
   | ShowNotification(notification)
   | HideNotification(int)
-  | SetExplorerTree([@opaque] UiTree.t)
-  | UpdateExplorerNode([@opaque] UiTree.t, [@opaque] UiTree.t)
+  | FileExplorer(FileExplorer.action)
```

Comment:
This is a good precedent for us to follow as we add more actions. I'll work on moving to a similar model for the language features I'm integrating.